packages/ui/tsconfig.esm.json

```diff
@@ -28,7 +28,7 @@
       "@testing-library/jest-dom",
     ]
   },
-  "include": ["src/public-api.ts", "src/models-api.ts", "src/testing-api.ts", "src/global.d.ts"],
+  "include": ["src/public-api.ts", "src/models-api.ts", "src/testing-api.ts","src/components-api.ts", "src/global.d.ts"],
```

Comment:
Should we add this on the other tsconfig.json as well?

Reply:
isn't that implicitly inherited by ` "extends": "./tsconfig.esm.json",` ?

Reply:
Yes, you're right. [Here's the extension](https://github.com/KaotoIO/kaoto/blob/c5545336f8304d91e11c62c9c3ed3c7ae09b9f25/packages/ui/tsconfig.cjs.json#L2)